linux/riscv64.config-patch

```diff
@@ -54,15 +54,15 @@
  # end of RISC-V CPU Idle Drivers
  # end of CPU Idle
  # end of CPU Power Management
-@@ -1737,6 +1744,7 @@
- CONFIG_PCI_STUB=y
- CONFIG_PCI_PF_STUB=m
- CONFIG_PCI_ATS=y
-+CONFIG_PCI_ECAM=y
```

Comment:
Is the removal of `+CONFIG_PCI_ECAM=y` intended?

Reply:
This is modified by `make olddefconfig` and seems almost unrelated to riscv.